modules/ingester/flush.go

```diff
@@ -261,8 +266,25 @@ func (i *Ingester) handleComplete(op *flushOp) (retry bool, err error) {
 	return false, nil
 }
 
-func (i *Ingester) handleFlush(userID string, blockID uuid.UUID) (retry bool, err error) {
-	level.Info(log.Logger).Log("msg", "flushing block", "userid", userID, "block", blockID.String())
+// withSpan adds traceID to a logger, if span is sampled
+// TODO: move into some central trace/log package
```

Comment:
Cortex has this - https://github.com/cortexproject/cortex/blob/master/pkg/util/spanlogger/spanlogger.go. Might be worth just using the same